docs/faq.md

```diff
@@ -471,6 +471,7 @@ If the registry returns `401 Unauthorized` or `"code":"UNAUTHORIZED"`, it is oft
       - Configuring credential helpers: [`<from/to><credHelper>`](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin#using-docker-credential-helpers) for Maven / [`from/to.credHelper`](https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin#using-docker-credential-helpers) for Gradle
       - Specific credentials (not recommend): [`<from/to><auth><username>/<password>`](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin#using-specific-credentials) or in [`settings.xml`](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin#using-maven-settings) for Maven / [`from/to.auth.username/password`](https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin#using-specific-credentials) for Gradle
       - These parameters can also be set through properties: [Maven](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin#system-properties) / [Gradle](https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin#system-properties)
+* You previous used the deprecated `gcloud docker` to configure the `auths` block in `config.json`. Those credentials expire and need to be removed. Edit your `config.json` and remove `auths` associated with `gcr.io` (for example: `"https://asia.gcr.io"`). You can run `gcloud auth configure-docker` to correctly configure the `credHelpers` block for interactions with Google Container Registry.
```

Comment:
"You previously ..."

I'd like to put "short-lived" into "Those credentials expire ..." to emphasize it expires very soon. (An hour or two, isn't it?)